docs/4.x/tracking-java.md

```diff
@@ -0,0 +1,169 @@
+---
+category: API Reference
+---
+# Java Tracker
+
+The Matomo Java Tracker is a Java library to send tracking data to Matomo servers. It is compatible with Java 1.8 and above. You can use it to track data from any Java application, including Java web applications, Java desktop applications, Java command line applications, Java batch applications, Java microservices and more.
+
+Due to the nature of Java applications, the API is different from the JavaScript Tracker API. Nevertheless it supports synchronous and asynchronous tracking, custom dimensions, custom variables, custom events, content tracking, ecommerce tracking, goal tracking, site search tracking, campaign tracking, and more.
+
+The primary source to look for an API documentation is the [JavaDoc](https://matomo-org.github.io/matomo-java-tracker/javadoc/index.html). This guide provides a quick overview of the most important classes and methods of the library.
+
+If you experience any problems or have any questions, please [create an issue](https://github.com/matomo-org/matomo-java-tracker/issues/new) in the [Matomo Java Tracker repository]. We are happy to help you.
```

Comment:
Should `[Matomo Java Tracker repository]` be linked? Or should the brackets be removed again?

Reply:
Fixed that.